sdk/cosmos/.azure-pipelines/cosmos.test.yml

```diff
@@ -0,0 +1,48 @@
+jobs:
```

Comment:
@bryevdv are you planning to create pipelines for these in our internal DevOps team after this merges? Or is that something you want our assistance with?

Reply:
I would love to pair with someone to do that

Reply:
I'm happy to pair with you on that.